editor.planx.uk/src/components/TestEnvironmentBanner.tsx

```diff
@@ -17,37 +17,33 @@ const TestEnvironmentWarning = styled(Box)(({ theme }) => ({
 }));
 
 const TestEnvironmentBanner: React.FC = () => {
-  const [showWarning, setShowWarning] = useState(true);
+  const [isTestEnvBannerVisible, hideTestEnvBanner] = useStore(state => [state.isTestEnvBannerVisible, state.hideTestEnvBanner]);
```

Comment:
This is now controlled at a store level as it was being re-rendered (and therefore going back to visible) on each route change.

I think this is the same issue we're seeing with everything re-rendering when we got to `/nodes` routes. See comment below on this 👍